screencastr.thor

```diff
@@ -13,12 +14,21 @@ class Screencastr < Thor
     # a. Will need a naming scheme to match the location properly
 
   desc "add_bumpers VIDEO_PATH", "Add bumpers to video"
-  def add_bumpers(video)
+  def add_bumpers(video_path)
     video = FFMPEG::Movie.new(video_path)
   end
 
   desc "add_watermark VIDEO_PATH", "Add watermark to video"
-  def add_watermark(video)
+  def add_watermark(video_path)
     video = FFMPEG::Movie.new(video_path)
   end
+
+  desc "transcode VIDEO_PATH", "Transcode video file to mp4 format"
+  def transcode(video_path)
+    video = FFMPEG::Movie.new(video_path)
+    path_array = video_path.split('/')
```

Comment:
I think you can use the [`File` object in the Ruby standard library](https://ruby-doc.org/core-2.4.2/File.html) to more consistently parse out the parts of the path rather than doing it manually.

I might also consider moving that path and filename parsing out into it's own method since it will probably be reused by all of these other methods.